x-pack/test/functional/services/transform/wizard.ts

```diff
@@ -270,10 +270,48 @@ export function TransformWizardProvider({ getService }: FtrProviderContext) {
       );
     },
 
-    async addAggregationEntry(index: number, identifier: string, expectedLabel: string) {
+    async addAggregationEntry(
+      index: number,
+      identifier: string,
+      expectedLabel: string,
+      formData?: Record<string, any>
+    ) {
       await comboBox.set('transformAggregationSelection > comboBoxInput', identifier);
       await this.assertAggregationInputValue([]);
       await this.assertAggregationEntryExists(index, expectedLabel);
+
+      if (formData !== undefined) {
+        await this.fillPopoverForm(identifier, expectedLabel, formData);
+      }
+    },
+
+    async fillPopoverForm(
+      identifier: string,
+      expectedLabel: string,
+      formData: Record<string, any>
+    ) {
+      await testSubjects.existOrFail(`transformAggPopoverForm_${expectedLabel}`);
+
+      for (const [testObj, value] of Object.entries(formData)) {
+        switch (testObj) {
+          case 'filterAggTypeSelector':
+            await this.selectFilerAggType(value);
+            break;
+          case 'filterTermValueSelector':
+            await this.fillFilterTermValue(value);
+            break;
+        }
+      }
+      await testSubjects.clickWhenNotDisabled('applyTransformAggChanges');
```

Comment:
Nit: We tried to prefix/namespace all transform related `data-test-subj` with `transform` (except some nested ones). So this one could be something like `transformApplyAggChanges`.

Reply:
Renamed in https://github.com/elastic/kibana/pull/67591/commits/a9723ede4749a99674ab377327d07cd1ea9cbe0d